### ISSUE TITLE
Salexyun

### ISSUE THREAD

**Issue by minkull** [researchart/rose6icse#145]
https://github.com/researchart/rose6icse/tree/master/submissions/reproduced/salexyun

Authors of reproduction:
Rahul N. Iyer, S. Alex Yun, Meiyappan Nagappan, and J. Hoey

Github ids:
- rahul-iyer
- salexyun, corresponding author
- meido
- jessehoey

Authors of original paper:
J. Tsay, L. Dabbish, and J. Herbsleb

# Authors applying for reproduced badge

**Comment by artifact-review** [comment]
The authors apply for "Reproduced" badge

+ **Available**: Both the filtered data from original paper and new data supplemented from RepoReaper are available on Dropbox. Also, the R source code to run linear mixed-effect models are also available on the same Dropbox folder. Some suggestions from reviewer: 
1. The author might consider using other hosting services for 
2. Separate data from Tsay et al' work and RepoReaper

+ **Reproduced**: The data and reproducing process are relatively straightforward and work as expected. 

**Decision:** Accepted

**Comment by timm** [comment]
@anonymousICSEartifacts : looking forward to your comment.

**Comment by timm** [comment]
@anonymousICSEartifacts : looking forward to your comment.

**Comment by ghost** [comment]
- **Functional.** Artefact is documented, complete, exercisable, and includes appropriate evidence. 

- **Available.** The artefact is placed on dropbox.

- **Reproduced.** The main results of the paper have been obtained by other researchers. However, according to the authors, they have used 11,000 projects that were used in the previous paper (although they have added more). Therefore, there seems to be some kind of dependency between both studies.

For this reason, I think that perhaps the recommendation should be replicated instead of reproduced. Although their study is a mixture of both (their set of projects is composed by new projects and most of the previous projects). Out of the 12,482 projects used in the original study, the authors use 11,000, and have added 1,860 more.

**Comment by timm** [comment]
@rahul-iyer
@salexyun, corresponding author
@meido
@jessehoey

Everyone agrees with  @replicated 

@reproduced is an issue. can the authors offer any reply to @anonymousICSEartifacts concerns?

**Comment by ghost** [comment]
I agree with replicated. Waiting for authors' explanations about reproduced.

**Comment by salexyun** [comment]
Thank you for all your comments. We believe our work is better represented as reproduced than replicated. Although we used the same 11,000 projects used in Tsay et al.'s, the data was recollected and up to a more recent date.

Regarding one of the reveiwer's comment: "Out of the 12,482 projects used in the original study, the authors use 11,000, and have added 1,860 more." This is not quite accurate. We combined the 11,000 projects from Tsay et al.'s and 15,000 projects from _RepoReaper. We then filtered a total of 26,000 projects down to 1,860 projects by only considering those projects that have at least 250 closed or merged pull requests.

In other words, we did not simply take the exact same dataset as Tasy et al.'s. We reproduced their approach on the latest projects and collected more up-to-date data for each project, and also added significant amount of new projects. Hence, we claim reproduced.

**Comment by timm** [comment]
I am now leaning towards "reproduced" 

BUT

I would concur with a veto from @anonymousICSEartifacts or @artifact-review if they see fit to issue one

**Comment by ghost** [comment]
The ACM "Reproduced" badge  is defined as:

- The main results of the paper have been independently obtained in a subsequent study by a person or team other than the authors, without the use of author-supplied artifacts.

In this case, the results have not been totally independently obtained. As clarified by the authors, they have "reproduced their approach on the latest projects". I understand this implies they have followed the same process the original researchers have. This is a dependency.

**Comment by ghost** [comment]
With this I mean that I still think the study is "replicated". As long as there is nothing between "replicated" and "reproduced", and the total independency required by reproduced is not met.

@timm, am I misinterpreting the "reproduced" badge?

**Comment by minkull** [comment]
Hi @anonymousICSEartifacts 

Reproducibility is frequently obtained by using the same methodology, but independently. So, it is ok that the same process was used as in the original paper, so long as the authors have not used the artifacts from the original study. For instance, they should not use scripts for data collection provided in the original paper.

Based on the above, are you ok with the "reproduced" badge?

**Comment by ghost** [comment]
Thanks for your clarification, Leandro. Yes, then I am fine with "reproduced".

**Comment by salexyun** [comment]
We can make further clarification:
- Tsay provided the data, which we used to get the list of the repositories
- Using our own script, we recollected all the data (dated ~May, 2018)
- Tsay provided the script for building the model; hence, we used the same model

**Comment by minkull** [comment]
Many thanks everyone for all the discussions!

Reproduced badge awarded.
